src/Calendar.jsx

```diff
@@ -499,8 +502,8 @@ export default class Calendar extends Component {
           selectRange && valueArray.length === 1 && 'react-calendar--selectRange',
           className,
         )}
-        onMouseOut={selectRange ? onMouseOut : null}
-        onBlur={selectRange ? onMouseOut : null}
+        onMouseLeave={selectRange ? onMouseLeave : null}
```

Comment:
nice catch! 🚫unnecessary re-renders 🚫

Reply:
Actually this changes behavior a little bit. We're no longer removing hover highlight when the cursor moves from tiles to weekday names / top navigation. What we should do instead to keep the current behavior is add onMouseLeave to MonthView, YearView, DecadeView and CenturyView.